### PR TITLE
Adding in the Front end for Machine to Machine

### PR DESCRIPTION
**Pull request by DanielJMaher**

### CODE REVIEW

**Comment by DanielJMaher** [comment]
@FBRTMaka @oceanzus

File: ooiui/core/routes/m2m.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+import requests
+import urllib2
+
+#M2M Interface
+
+@app.route('/api/m2m/get_data/<string:reference_designator>/<string:method>/<string:stream>/<string:start>/<string:end>')
+def get_data(reference_designator, method, stream, start, end):
+    token = get_login()
+    url = app.config['SERVICES_URL'] + '/m2m/get_data/%s/%s/%s/%s/%s' % (reference_designator, method, stream, start, end)
+    print url
```
Comment (FBRTMaka) [inline ooiui/core/routes/m2m.py]: @DanielJMaher print

File: ooiui/core/routes/m2m.py [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    for each in request.args:
+        print each, request.args[each]
+    response = requests.get(url, auth=(token, ''), params=request.args)
+    print response.text
```
Comment (FBRTMaka) [inline ooiui/core/routes/m2m.py]: @DanielJMaher blah blah

**Comment by FBRTMaka** [comment]
@DanielJMaher good nuf for ooi!

**Pull request merged**